Docs: agregar resumen del capitulo 4

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 4 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html
- https://github.com/accesibleprogramacion/ahorcado/blob/main/script.js

IIKUYY committed:
Docs: agregar resumen del capitulo 4

## what

- Agregar el resumen del capitulo 4 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html
- https://github.com/accesibleprogramacion/ahorcado/blob/main/script.js

semanticmx commented on Ch4/Canvas/Canvas.html:
```diff
@@ -0,0 +1,17 @@
+    <head>
+        <title>Canvas</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
```
hay que empezar a listar los atributos en una sola columna, en lugar de una sola fila cuando son más de uno por tag.

en lugar de:

```html
<meta name="viewport" content="width=device-width, initial-scale=1">
```

hacer:

```html
<meta 
  name="viewport" 
  content="width=device-width, initial-scale=1"
>
```

opcional, colocarlos en orden alfabético

hacer:

```html
<meta 
  content="width=device-width, initial-scale=1"
  name="viewport" 
>
```

semanticmx commented on Ch4/Canvas/Java.js:
```diff
@@ -0,0 +1,5 @@
+function drawB() {
```
los nombres de archivos CSS y JS "default" se pueden llamar main.css o main.js

para HTLM, los nombres de los archivos hay que cambiarlos a todos en minúsculas y usando guión medio.

ejemplo:

canvas.html
java.js
main.css
main.js
contact-us.html

README.md no es un set de extensiones relacionadas a HTML por lo que ese queda en mayúsculas con la extensión en minúsculas.

Y en lugar de Java hay que usar Javascript porque son lenguajes con usos diferentes.

los archivos JS pueden llamarse javascript.js pero cuando agreguemos más idealmente queremos intuir el contenido de los archivos desde el mismo nombre del archivo, por ejemplo:

main.js
plugins.js
components.js
services.js
utils.js

Todos los nombres de archivo anteriores te dan una idea de lo que se almacena en cada uno.

Para este capítulo dado que solo se utiliza un archivo por HTML todos se pueden llamar main.js

semanticmx commented on Ch4/Coordenadas/Coordenadas.html:
```diff
@@ -0,0 +1,29 @@
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <style>
+        .canvas{
```
el contenido también identalo para que quede así:

```html
<style>
         .canvas{
         ...
</style>
```

semanticmx commented on Ch4/README.md:
```diff
@@ -0,0 +1,11 @@
+# [Una superficie para dibujar](https://github.com/IIKUYY/HTML5/tree/main/Ch4/Ch4.md)
```
Puedes agregar un nuevo folder dentro de Ch4 que se llame assets.

Dentro de ese folder puedes agregar screenshots de lo que generaste con canva

Y en el readme puedes agregar una o varias imágenes por cada subtema para tener una referencia visual de lo que hace cáda código.

semanticmx reviewed (changes requested):
muy bien, ya está mejor indentado y con mejor uso de camelCase.

IIKUYY committed:
Chores: Realizar cambios necesarios

## what

- Cambiar nombres de archivos
- Corregir la identación
- Agregar capturas de pantalla junto a la carpeta assets
- Corregir semántica de HTML

## why

- Arreglar la semántica de los archivos
- Corregir error en los nombres
- Mejorar calidad de lectura del repositorio

## references
-

IIKUYY commented:
Se realizar los cambios pedidos

semanticmx reviewed (approved):
gracias!
